libs/ui/src/lib/static-autocomplete/group/group.component.ts

```diff
@@ -100,14 +100,18 @@ export class GroupMultiselectComponent implements OnInit, OnDestroy {
   }
 
   getSelectable(groups: StaticGroup[], filter?: string): Record<string, boolean> {
+    const scope = this.scope === 'territories' ? territories : medias;
```

Comment:
this makes the component "binded" to territories or medias.
```suggestion
    const scope = staticModel[this.scope];
```